doc/aws-deployment.md

```diff
@@ -0,0 +1,386 @@
+need.  So when creating a new deployment we suggest creating a new branch (based
+off of `develop`) and updating the files which require different values—see step
+3 below.
+
+1.  Copy the `update-from-github.sh` to the `/home/build` directory (make it
+    executable).
+
+2.  Set the `BRANCH_TAG` value in the script to identify the
+    `nhsd-datadictionary-docker` branch to use on this server—make sure the
+    branch has been pushed).
+
+3.  If you are deploying to a new server (with a new hostname) check the
+    `up.sh` (`down.sh`?) and `.env` files to ensure that they are exporting the
+    correct value for `MAURO_ENDPOINT`.
+
+4.  Run the `update-from-github.sh` script.  With all the above in place the
+    site should be available via the hostname
+
+5.  Work through the "Configuring Mauro" section of the Site setup
+    documentation.
+
+6.  Ingest a suitable branch and when that has completed create a new version
+
+\pagebreak
+# Configuring Mauro
+
+With the site running there are a number of tasks to carry out before it can
+be used.
+
+## Administrator account
+
+When you first access Mauro (MDM UI, not the Orchestrator), you can sign in
+with the standard Mauro account created during bootstrap:
+
+Username: "admin@maurodatamapper.com"
+Password: "password"
+
+After sign in, you will be forced to change this password to something more
+secure.
+
+## Site URL
+
+There is a configuration token used in Mauro, particularly for email templates,
+stating what the base URL is for this Mauro instance. This must be set
+manually.
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Configuration"
+3.  Click the "Add" button
+4.  Under "Select the property to add", select "site.url"
+5.  Under "Value", enter the base URL domain this Mauro instance uses.
+6.  Click "Add property" to save.
+
+## OpenID Connect
+
+The OpenID Connect feature needs to be switched on in Mauro.
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Configuration"
+3.  Click the "Add" button
+4.  Under "Select the property to add", select "feature.use_open_id_connect"
+5.  Under "Value", select "Yes".
+6.  Click "Add property" to save.
+
+## User groups
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Manage groups"
+3.  Click the "Add" button.
+4.  Create a new group called "readers", then click "Add group"
+5.  Repeat for a new group called "editors".
+
+## User accounts
+
+Once groups are added, create the user accounts.
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Manage users"
+3.  Click "Add"
+4.  Enter the details for each user (see below), including the user group they
+    belong in.
+5.  Click "Add user" to save.
+6.  Repeat for each user.
+
+**Note**: at this point in the setup, you may not have SMTP setup yet so the
```

Comment:
You mention that SMTP might not be set up, but we haven't got any instructions for configuring SMTP when they do use it. These environment variables for the Docker build scripts should be explained so they can do it themselves.

Reply:
Add a note to the appendix